aviary/mission/height_energy_problem_configurator.py

```diff
@@ -313,13 +313,6 @@ def link_phases(self, aviary_group, phases, connect_directly=True):
                 phase.set_state_options(Dynamic.Vehicle.MASS, input_initial=False)
                 phase.set_state_options(Dynamic.Mission.DISTANCE, input_initial=False)
 
-        aviary_group.connect(
```

Comment:
This capability was moved into the state_output subsystem in AviaryGroup because it was needed in all cases.